AnaProd/tasks.py

```diff
@@ -94,8 +97,8 @@ def getInputFiles(central_nanoAOD_path, sample_name):
         return list(sorted(input_files))
 
     @staticmethod
-    def getOutputFile(central_anaTuples_path, sample_name, split_idx):
-        return os.path.join(central_anaTuples_path, '_pre', sample_name, f'anaTuple_{split_idx}.root')
+    def getOutputDir(central_anaTuples_path, sample_name):
+        return os.path.join(central_anaTuples_path, '_pre', sample_name)
```

Comment:
Remove `_pre` from the output dir. Now these are final anaTuples

Reply:
ok